### PR TITLE
fix: Remove dead verdict code from evaluate(), add validate() guard (ADV-0067)

### PR DESCRIPTION
**Pull request by movito**
## Summary
- Remove ~55 lines of dead/duplicate verdict extraction code from `cli.evaluate()` — it re-parsed verdicts already handled by `run_evaluator()`
- `evaluate()` now follows the same trust-`run_evaluator()` pattern as `review()` (consistency)
- Add empty `test_command` guard in `validate()` to prevent `IndexError` on `""` or `"  "` input
- Tests updated: removed mocks for deleted code paths, added 4 new `validate()` edge case tests

## Test plan
- [x] 544 tests passing, 0 failures
- [x] `evaluate()` success path works without re-parsing verdict
- [x] `evaluate()` failure path returns non-zero from `run_evaluator()`
- [x] `validate("")` returns 1 with error message (no IndexError)
- [x] `validate("  ")` returns 1 with error message
- [x] `validate(None)` falls back to config default
- [x] Pattern lint clean
- [x] CI checks pass locally

🤖 Generated with [Claude Code](https://claude.com/claude-code)

---

> [!NOTE]
> **Low Risk**
> Low risk cleanup and defensive checks in CLI flow; main impact is slightly different success messaging and earlier failure for empty `test_command` inputs.
> 
> **Overview**
> **Simplifies plan evaluation output flow.** `cli.evaluate()` no longer re-opens logs or re-parses verdicts/token counts after `run_evaluator()` succeeds; it now relies entirely on `run_evaluator()`’s exit code and prints a single success message.
> 
> **Adds defensive validation for test execution.** `cli.validate()` now errors out early when `test_command` is empty/whitespace to avoid crashing on `shlex.split()`/`subprocess.run`, and `cli.review()` messaging is aligned to “complete”. Tests were updated to remove coverage for the deleted helpers and add `validate()` edge-case assertions.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 28c2010a33f411e9b9e01a990ecc639a09e3f9c1. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: adversarial_workflow/cli.py [hunk 1]
```diff
@@ -1731,67 +1731,9 @@ def evaluate(task_file: str) -> int:
...
-        print(f"{RED}❌ Evaluation REJECTED{RESET}")
-        print("   Plan has fundamental issues - major revision needed")
-        print(f"   Details: {log_file}")
-        return 1
-    else:  # UNKNOWN or other
-        print(f"{YELLOW}⚠️  Evaluation complete (verdict: {verdict}){RESET}")
-        print(f"   Review output: {log_file}")
-        return 0
+    print(f"{GREEN}✅ Evaluation approved!{RESET}")
+    return 0
```
[the remark this answers is not in the document]
Comment (movito) [reply adversarial_workflow/cli.py:1567]: Fixed in 7f809be. Changed to neutral "✅ Evaluation complete\!" / "✅ Review complete\!" messages that are accurate regardless of which zero-exit-code path run_evaluator() took (APPROVED vs UNKNOWN vs cancelled). Good catch.

[the remark this answers is not in the document]
Comment (movito) [reply adversarial_workflow/cli.py:1567]: Fixed in 7f809be. Changed to neutral "✅ Evaluation complete\!" / "✅ Review complete\!" messages. Also applied the same fix to review() which had the identical issue. Good catch.

File: adversarial_workflow/cli.py [hunk 2]
```diff
@@ -1731,67 +1731,9 @@ def evaluate(task_file: str) -> int:
...
-        print(f"{RED}❌ Evaluation REJECTED{RESET}")
-        print("   Plan has fundamental issues - major revision needed")
-        print(f"   Details: {log_file}")
-        return 1
-    else:  # UNKNOWN or other
-        print(f"{YELLOW}⚠️  Evaluation complete (verdict: {verdict}){RESET}")
-        print(f"   Review output: {log_file}")
-        return 0
+    print(f"{GREEN}✅ Evaluation complete!{RESET}")
+    return 0
```
[the remark this answers is not in the document]
Comment (movito) [reply adversarial_workflow/cli.py:1567]: Fixed in 28c2010. Removed all 4 orphaned functions (verify_token_count, estimate_file_tokens, extract_token_count_from_log, validate_evaluation_output) and their tests. Also removed the now-unused `import re`.

[the remark this answers is not in the document]
Comment (movito) [reply adversarial_workflow/cli.py:1567]: Fixed in 28c2010. Removed the duplicate validate_evaluation_output from cli.py. The canonical version in utils/validation.py (used by runner.py) remains, with full test coverage in test_utils_validation.py.
